.github/workflows/release-to-pypi.yml

```diff
@@ -0,0 +1,80 @@
+name: Release to PyPI
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build-wheels:
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+
+    - name: Check out the release commit
+      uses: actions/checkout@v2
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: arm64
+      if: runner.os == 'Linux'
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install Python packages needed for wheel build and upload
+      run: |
+        python -m pip install --upgrade pip setuptools
+        python -m pip install twine wheel
+
+    - name: Build wheels
+      uses: pypa/cibuildwheel@v2.9.0
+      env:
+        CIBW_SKIP: 'pp*'
+        CIBW_ARCHS_LINUX: "auto aarch64"
+        CIBW_ARCHS_MACOS: "auto universal2"
+
+    - name: Check and upload wheels
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python -m twine check --strict wheelhouse/*.whl
+        python -m twine upload wheelhouse/*.whl
+
+  build-sdist:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Check out the release commit
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install Python packages needed for sdist build and upload
+      run: |
+        python -m pip install --upgrade pip setuptools
```

Comment:
If you use `python -m build` to build as suggested below, then `setuptools` shouldn't be needed here.